npm run build: Module not found: Error: Can't resolve 'original-fs' in '[...]/vscode-firefox-debug/node_modules/adm-zip/util'

Module not found: Error: Can't resolve 'original-fs' in '/my/path/vscode-firefox-debug/node_modules/adm-zip/util'

I did: 
`npm i --save original-fs`
npm run build succeded.

now i get from: `npm run package`:
` ERROR  filePath.endsWith is not a function`

node --version: v19.6.0
npm --version: 9.4.2

--- a/package.json
+++ b/package.json
@@ -75,6 +75,7 @@
     "gulp-uglify": "^3.0.2",
     "mocha": "^10.2.0",
     "nyc": "^15.1.0",
+    "original-fs": "^1.2.0",
     "rimraf": "^5.0.1",
     "ts-node": "^10.9.1",
     "typescript": "^5.2.2",

